index.html

```diff
@@ -172,7 +172,8 @@ <h1 class="section-header">How to Use</h1>
       <i class="fa fa-chevron-circle-up" aria-hidden="true"></i>
     </span>
     <footer>
-      <span class="tagline">Made with <span class="tagline-heart">❤︎</span></span><br />
+      <span class="tagline">Made with <span class="tagline-heart">❤︎</span></span><br/>
+      Say Hello! to our awesome <a href="https://github.com/shahednasser/sbuttons/blob/master/CONTRIBUTORS.md">contributors</a><br/>
```

Comment:
Can you move the exclamation mark to the end of the sentence? (Basically after contributors)